_content/applications/tago/index.md

```diff
@@ -4,36 +4,45 @@ section: Integrations
 source: 'https://github.com/TheThingsNetwork/docs/blob/master/_content/applications/tago/index.md'
 ---
 
-# Tago
+# TagoIO
 
-[Tago](https://tago.io/) allows you to quickly store, visualize, and act on sensor data.
+Here you will learn how to integrate TTN LoRaWAN devices with [TagoIO](https://tago.io/). You will be able to send data from your devices (uplink) to TagoIO and back to them (downlink).
 
-After creating an account on [Tago.io](https://tago.io/), activate the integration in The Things Network [Developer Console](https://console.thethingsnetwork.org/) by going to your application and Select Tago from the integration options.
 
-![Add-integration](Tago1.png)
+## 1. TagoIO Setup
+After creating an account on [Tago.io](https://tago.io/), go to Devices, click on *Add Devices*, and filter by LoRaWAN TTN. Then, search for the device that you want to add. If you don't find it, select the type *Custom TTN Device*.
```

Comment:
Try and avoid using too many abbreviations like "don't". This is a formal technical document, not a casual letter to a friend.